### ISSUE TITLE
Option to change base "local" folder

### ISSUE THREAD

**Issue by yphastos** [Natizyskunk/vscode-sftp#180]
**Is this a similar or duplicate feature request?**
- [ ] I don't know. I will go check it.
- [x] No.

**Is your feature request related to a problem? Please describe.**
If I wrongly choose the initial folder upon installing the extension, or I want to change it at any time, I can't easily change it.
I couldn't find a setting or option where I can do so.

**Describe the solution you'd like**
An option or setting to be able to change the "base" folder for where all local files are stored.

**Describe alternatives you've considered**
The only way I could do so was uninstalling the extension and reinstalling it again, so I get prompted to choose the base folder.
Maybe the option is already there somewhere, but couldn't find it.

**Does this project help you?**
- [x] Yes. SFTP IS AWESOME!

**Comment by Natizyskunk** [comment]
@yphastos,
Can you be more precise on what you want exactly and how you achived your current state please ?
Maybe some screenshots could help :)

**Comment by yphastos** [comment]
@Natizyskunk 
Hello. I'll try to be more specific.

When we first run the extension, we are prompted with an option to choose a "working folder", like this:

![image](https://user-images.githubusercontent.com/13773388/189768408-d4012bc2-9136-4c35-953f-c47827a4f16e.png)

From there, we choose the "local" folder, which is the one all our remote files will be saved "locally", it also creates a `.vscode` subfolder, which contains the `sftp.json`, and we can then edit such file to give our server data, etc.

As an example, in my case, I choose the "local" folder, as:
`D:\Dev\vsCode SFTP\`

Then I made a couple subfolders, for different remote (server side) folders, along with the `.vscode` one, e.g.

![image](https://user-images.githubusercontent.com/13773388/189769326-3d528427-0dff-4bb1-9593-2a06020409fc.png)

And the corresponding settings file:

![image](https://user-images.githubusercontent.com/13773388/189770248-e114f8ae-34b6-4e3f-86f4-ad2476114e28.png)

(I use the context option to have several folders)
This works perfectly, and I can work with files locally, and upload changes to the remote side of each folder.

But my question is, what if I want to change the "local" folder, (that is, `D:\Dev\vsCode SFTP\` in my case), for something else?
e.g. either if I initially messed up choosing a wrong folder, or I just want to change where my local files are stored.

In other words, where can I change the `D:\Dev\vsCode SFTP\` path to something else?

I couldn't find anywhere on the settings where such path is stored, so I couldn't figure out where to change it.
Or even a way to "trigger" the initial "open folder" dialog I've shown at the beginning of this post.

I hope it's more clear now.

Thanks in advance.

**Comment by Natizyskunk** [comment]
@yphastos,
You shoudn't have one folder with one sftp.json config file and all your rpoject in it as subfolders.
You should preferably have an `sftp.json` config file in your `.vscode/` folder for each project you want to use the extension with.
> one project = one `sftp.json` config file.
